### DISCUSSION TITLE
Undocumented basic information - Coordinate conventions?

### DISCUSSION THREAD

**Discussion by lucem** [DiligentGraphics/DiligentEngine#281] [Q&A] [answered]
Hi everyone, I am puzzled, because I cannot seem to find the most basic information about Diligent in the documentation or the tutorials and examples.

I plan to use Diligent for code-generated rendering, no imported models (yes, it is voxels...)
However, in order to spare myself wasting some hours on writing experiments to find out -

What is the coordinate system convention in Diligent? Left-handed or right handed? Y up or Z up?
What is the triangle winding order? Clockwise or counterclockwise?

And, while we're at it, please add this information into the documentation!

Thank you!

**Comment by MikhailGorobets** [comment, marked as answer]
@lucem 
https://github.com/DiligentGraphics/DiligentCore/blob/master/doc/CoordinateSystem.md

**Reply by lucem** [reply]
Thank you, so it has been there all along, and I just couldn't find it!
